java/cdk/src/main/java/sleeper/cdk/stack/QueryStack.java

```diff
@@ -247,47 +251,49 @@ private void attachPolicy(IFunction lambda, String id) {
      * @param  instanceProperties containing configuration details
      * @return                    the queue to be used for leaf partition queries
      */
-    private Queue setupLeafPartitionQueryQueue(InstanceProperties instanceProperties) {
+    private Queue setupLeafPartitionQueryQueue(InstanceProperties instanceProperties, Topic topic) {
         String dlLeafPartitionQueueName = Utils.truncateTo64Characters(instanceProperties.get(ID) + "-LeafPartitionQueryDLQ");
-        Queue leafPartitionQueryQueueForDLs = Queue.Builder
-                .create(this, "LeafPartitionQueriesDeadLetterQueue")
+        Queue leafPartitionQueryDlq = Queue.Builder
+                .create(this, "LeafPartitionQueryDeadLetterQueue")
                 .queueName(dlLeafPartitionQueueName)
                 .build();
-        DeadLetterQueue leafPartitionQueriesDeadLetterQueue = DeadLetterQueue.builder()
+        DeadLetterQueue leafPartitionQueryDeadLetterQueue = DeadLetterQueue.builder()
                 .maxReceiveCount(1)
-                .queue(leafPartitionQueryQueueForDLs)
+                .queue(leafPartitionQueryDlq)
                 .build();
-        String leafPartitionQueueName = Utils.truncateTo64Characters(instanceProperties.get(ID) + "-LeafPartitionQueriesQueue");
-        Queue leafPartitionQueriesQueue = Queue.Builder
-                .create(this, "LeafPartitionQueriesQueue")
+        String leafPartitionQueueName = Utils.truncateTo64Characters(instanceProperties.get(ID) + "-LeafPartitionQueryQueue");
+        Queue leafPartitionQueryQueue = Queue.Builder
+                .create(this, "LeafPartitionQueryQueue")
```

Comment:
Please can we keep the queue names and CDK IDs consistent between this and the main query queue? Either query or queries is fine, as long as it's the same for both.